core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
+### Subject Key Identifier (SKI)
+### Authority Key Identifier (AKI)
+
+The concatenation of ´SKI´ and ´AKI´ provides a unique identifier valid - even if multiple
+ CAs are able to issue valid certificates.
+
+EXAMPLE (snippet from a X.509 certificate):
+
+```
+...
+
+X509v3 extensions:
+    X509v3 Subject Key Identifier:
+        DD:CB:FD:0B:93:84:33:01:11:EB:5D:94:94:88:BE:78:7D:57:FC:4A
+    X509v3 Authority Key Identifier:
+        keyid:CB:8C:C7:B6:85:79:A8:23:A6:CB:15:AB:17:50:2F:E6:65:43:5D:E8
+
+...
+```
+
+... leads to a [connectors](../Connector/README.md) `unique identifier`:
+
+```
+DD:CB:FD:0B:93:84:33:01:11:EB:5D:94:94:88:BE:78:7D:57:FC:4A:keyid:CB:8C:C7:B6:85:79:A8:23:A6:CB:15:AB:17:50:2F:E6:65:43:5D:E8
+```
+
+In examples and for reasons of readability editors might use
+
+```
+SKI:AKI
+```
+
+See also:
+- [Glossary "Dynamic Attribute Token"](../../glossary/README.md#dynamic-attribute-token)
+- Shortcut: `DAT`
+- [X.509 certificates](https://en.wikipedia.org/wiki/X.509)
+
 --- 
 
+
+## Request token that is handed in at DAPS side
+
+|**Field name**|**mandantory**|**cardinality**|**content**
+|:---|:---|:---|:---|
+|**`@context`**  | yes   | 1 | The JSON-LD context containing the IDS classes, properties and instances. Must be "https://w3id.org/idsa/contexts/context.jsonld". |
+|**`@type`**     | yes   | 1 | In the context of the IDS, the request payload is an RDF instance and therefore must state that it has "@type" : "ids:DatRequestToken". |
+|**`sub`**       | yes   | 1 | Subject the requesting connector the token is created for. This is the connector requesting the DAT. The "sub" value must be the  combined entry of the SKI and AKI of the IDS X509 as presented in Sec. 4.2.1.  In this context, this is identical to "iss". |
+|**`exp`**       | yes   | 1 | Expiration date of the token. Can be chosen freely but should be limited to a short period of time (e.g., one minute). |
+|**`iat`**       | yes   | 1 | Timestamp the token has been issued. |
+|**`nbf`**       | yes   | 1 | "Valid not before": For practical reasons this should be identical to iat. If the system time is not in synch with the DAPS, the request token will be rejected (e.g., nbf is in the future). |
+|**`aud`**       | yes   | 1 | The audience of the token. This can limit the validity for certain connectors. |
+|**`iss`**       | yes   | 1 | According to RFC 7519 Sec. 4.1.1, the issuer is the component which created and signed the JWT. In the context of the IDS, this must be a valid connector. The "iss" value must be the combined entry of the SKI and AKI of the Connectors X509 certificate as presented in Sec. 4.2.1. |
+
+
+
+## Request call to get a token
+
+`client_assertion` id the base64 encoded request token, shown under
+ ["Request token that is handed in at DAPS side"](#request-token-that-is-handed-in-at-daps-side).
+
+
+|**FormBody Attribute**|**content**
+|:---|:---|
+|**`grant_type`**  | OAuth based grant type. See [TODO : client_credentials grant](TODO). |
```

Comment:
TODO?!

Reply:
This should probably not be a TODO. The grant type is (usually) "client_credentials" (https://tools.ietf.org/html/rfc6749#section-4.4)

Reply:
TODO means TODO. DRAFT-branch!